gradio/pipelines_helpers.py

```diff
@@ -0,0 +1,535 @@
+            "preprocess": lambda i, c, m: {
+                "sequences": i,
+                "candidate_labels": c,
+                "multi_label": m,
+            },
+            "postprocess": lambda r: {
+                r["labels"][i]: r["scores"][i] for i in range(len(r["labels"]))
+            },
+        }
+        return pipeline_info
+
+    elif is_transformers_pipeline_type(pipeline, "DocumentQuestionAnsweringPipeline"):
+        pipeline_info = {
+            "inputs": [
+                components.Image(type="filepath", label="Input Document", render=False),
+                components.Textbox(label="Question", render=False),
+            ],
+            "outputs": components.Label(label="Label", render=False),
+            "preprocess": lambda img, q: {"image": img, "question": q},
+            "postprocess": lambda r: {i["answer"]: i["score"] for i in r},
+        }
+        return pipeline_info
+
+    elif is_transformers_pipeline_type(pipeline, "VisualQuestionAnsweringPipeline"):
+        pipeline_info = {
+            "inputs": [
+                components.Image(type="filepath", label="Input Image", render=False),
+                components.Textbox(label="Question", render=False),
+            ],
+            "outputs": components.Label(label="Score", render=False),
+            "preprocess": lambda img, q: {"image": img, "question": q},
+            "postprocess": lambda r: {i["answer"]: i["score"] for i in r},
+        }
+        return pipeline_info
+
+    elif is_transformers_pipeline_type(pipeline, "ImageToTextPipeline"):
+        pipeline_info = {
+            "inputs": components.Image(
+                type="filepath", label="Input Image", render=False
+            ),
+            "outputs": components.Textbox(label="Text", render=False),
+            "preprocess": lambda i: {"images": i},
+            "postprocess": lambda r: r[0]["generated_text"],
+        }
+        return pipeline_info
+
+    elif is_transformers_pipeline_type(pipeline, "ObjectDetectionPipeline"):
+        pipeline_info = {
+            "inputs": components.Image(
+                type="filepath", label="Input Image", render=False
+            ),
+            "outputs": components.AnnotatedImage(
+                label="Objects Detected", render=False
+            ),
+            "preprocess": lambda i: {"inputs": i},
+            "postprocess": lambda r, img: (
+                img,
+                [
+                    (
+                        (
+                            i["box"]["xmin"],
+                            i["box"]["ymin"],
+                            i["box"]["xmax"],
+                            i["box"]["ymax"],
+                        ),
+                        i["label"],
+                    )
+                    for i in r
+                ],
+            ),
+        }
+        return pipeline_info
+
+    else:
+        raise ValueError(f"Unsupported pipeline type: {type(pipeline)}")
```

Comment:
This should be unwrapped from the `else`.

```suggestion
    raise ValueError(f"Unsupported pipeline type: {type(pipeline)}")
```